client-templates/github-enterprise/accept.json.sample

```diff
@@ -42,6 +42,14 @@
           "path": "commits.*.modified.*",
           "value": "*req*.txt"
         },
+        {
```

Comment:
this file is missing some of the updates done in the github-com one, is that intentional?